client/src/App.js

```diff
@@ -1,20 +1,76 @@
-import React from "react";
+import React, {useState, useMemo, useEffect} from "react";
 import { MuiThemeProvider } from "@material-ui/core";
-import { BrowserRouter, Route } from "react-router-dom";
-
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import { Redirect } from "react-router";
+import Navbar from "./pages/navbar"
 import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
-
+//import LandingPage from "./pages/Landing";
+import SignUp from "./pages/SignUp";
+import Login from "./pages/Login";
+import Background from "./pages/Background";
+import Dashboard from "./pages/Dashboard"
+import Faq from "./pages/faq";
 import "./App.css";
+import { UserContext } from "./contexts/UserContext";
+import Blog from './pages/Blog';
+
+
 
 function App() {
+  const [user, setUser] = useState(null);
+
+  const value = useMemo(
+    () => ({
+      user: user,
+      setUser: setUser,
+      
+    }),
+    [user, setUser]
+  );
+
+  function handleErrors(response) {
+    if (!response.ok) {
+        throw Error(response.statusText);
+    }
+    return response;
+}
+
+  
+
+  useEffect(() => {
+    async function getUser() {
+      
+      fetch("/auth/verify")
+      .then(handleErrors)
+      .then(response => response.json())
+      .then((data) => {let user = data; setUser(user);})
+      .catch(function(error){ let user = "failed to fetch"; setUser(user)})
+    }
+    getUser();
+  
+  },[]);
+  
+
   return (
+    <UserContext.Provider value={value}>
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Route path="/" component={LandingPage} />
+        <Route exact path="/" render={() => <Redirect to="/signup" />} />
+        <Route path="/signup" component={SignUp} />
+        <Route path="/login" component={Login} />
+        <Route path="/background" component={Background} />
+        <Navbar/>
```

Comment:
Will the navbar show up at the bottom of the routes above ^ You may need to put the navbar into another route and prefix all the dashboard pages with /dashboard